src/commands/build.ts

```diff
@@ -121,8 +122,12 @@ export async function command(commandOptions: CommandOptions) {
     }
   }
 
+  let minifierService = await esbuildStartService(); // create esbuild process for minifying (there may or may not be one in plugins)
```

Comment:
Wanted to comment on the changes here a little more.

Currently, we have one `esbuild` service running in our plugin system. That’s responsible for handling Snowpack’s default builder, unless otherwise specified. I originally tried to use that same service, but ran into a major roadblock: that service that lives in the `createBuildPipeline()` function can be overridden. So it’s not guaranteed to be there.

So to play more nicely with all the different ways you can run Snowpack, we just create a minifier service outside, here, that runs only on build. While that can mean 2 esbuild services running, overall that seems preferable to loading terser.

It also seems preferable to enforcing that the plugin service always runs, as that at worst would conflict with other plugins; at best it would simply be useless and process nothing.